src/ntfs/read.rs

```diff
@@ -293,9 +293,11 @@ pub fn manage_stop_times(collections: &mut Collections, path: &path::Path) -> Re
             );
         }
         let datetime_estimated = stop_time.datetime_estimated.map_or_else(
-            || match collections.stop_points[stop_point_idx].stop_type {
-                StopType::Zone => true,
-                _ => false,
+            || {
+                matches!(
+                    collections.stop_points[stop_point_idx].stop_type,
+                    StopType::Zone
+                )
```

Comment:
`PartialEq` is already defined on `StopType` so how about doing this instead
```suggestion
            || collections.stop_points[stop_point_idx].stop_type == StopType::Zone
```